src/main/java/frc/team166/robot/RobotMap.java

```diff
@@ -55,10 +56,10 @@
         //changes Solenoid ports into integers
         public final static int LIFT_TRANSMISSION_A = 1;
         public final static int LIFT_TRANSMISSION_B = 2;
-        public final static int MANIPULATOR_SOLENOID_INNER_A = 3;
-        public final static int MANIPULATOR_SOLENOID_INNER_B = 4;
-        public final static int MANIPULATOR_SOLENOID_OUTER_A = 5;
-        public final static int MANIPULATOR_SOLENOID_OUTER_B = 6;
+        public static final int MANIPULATOR_SOLENOID_INNER_A = 3;
```

Comment:
why doesn't this follow the same style as other items here? Looks like other items are "public final static", not "public final static"